src/storage/index.ts

```diff
@@ -81,3 +81,7 @@ export const review = new Storage<[], Review>({ id: 'review' });
 export const campaigns = new Storage<[], Campaigns>({ id: 'campaigns' });
 
 export const cards = new Storage<[], Cards>({ id: 'cards' });
+
+export const identifier = new Storage<[], { identifier: string }>({
```

Comment:
You wanna store this in the keychain, otherwise (I believe) it would never persist an uninstall.